src/scripts/tbtcv2-rewards/rewards.ts

```diff
@@ -97,11 +95,16 @@ const prometheusAPIQuery = `${prometheusAPI}/query`
 // rewards interval dates.
 const offset = Math.floor(Date.now() / 1000) - endRewardsTimestamp
 
-type InstanceParams = {
-  upTimePercent: number
-  avgPreParams: number
-  version: any
-}
+const utils = new Utils(
+  prometheusAPI,
+  prometheusAPIQuery,
+  prometheusJob,
+  offset,
+  endRewardsTimestamp,
+  requiredUptime,
+  october17Timestamp,
```

Comment:
Note that all the code handling `october 17` rewards case was removed. Please update/copy from the official repo for the latest changes. See https://github.com/threshold-network/merkle-distribution/pull/89/commits/055f8499962eeb798f9532e4095f4590762358f6